dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayoutColumn.java

```diff
@@ -8,43 +8,35 @@
  * Created by Jonathan Gamba
  * Date: 10/25/12
  */
-public class TemplateLayoutColumn {
+public class TemplateLayoutColumn extends ContainerHolder {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)